slides/08-Platform-Hub-New/1060-WhatDoYouThink.md

```diff
@@ -0,0 +1,5 @@
+## We love feedback
+
+Please give us feedback, this is a realtively new application so there's going to be teething issues. However, we are driven to sorting as many of them as we can and improving how projects interact with ACP. The following is the best place to make new issues https://github.com/UKHomeOffice/application-container-platform-bau/issues
```

Comment:
We shouldn't be pointing to the BAU board directly (as mentioned in a previous comment).